gp-templates/locale.php

```diff
@@ -13,7 +13,13 @@
 gp_tmpl_header();
 ?>
 
-	<h2><?php printf( __( 'Active Projects translated to %s', 'glotpress' ), esc_html( $locale->english_name ) ); ?></h2>
+	<h2><?php printf( __( 'Active Projects translated to %s', 'glotpress' ), esc_html( $locale->english_name ) ); ?>
+		<?php if ( $locale_glossary ) : ?>
+			<a href="<?php echo gp_url_join( '/languages', $locale->slug, 'default', 'glossary' ); ?>" class="glossary-link">Locale Glossary</a>
```

Comment:
'/languages' is incorrect here, you need to use the `gp_public_url()` call instead so that if GP is not installed in the root it links to the right place.